src/main/scala-2.11/com/linagora/gatling/imap/protocol/command/StoreHandler.scala

```diff
@@ -0,0 +1,67 @@
+package com.linagora.gatling.imap.protocol.command
+
+import java.util
+
+import akka.actor.{ActorRef, Props}
+import com.lafaspot.imapnio.client.IMAPSession
+import com.lafaspot.imapnio.listener.IMAPCommandListener
+import com.linagora.gatling.imap.protocol.{Command, ImapResponses, Response, Tag}
+import com.sun.mail.imap.protocol.IMAPResponse
+import io.gatling.core.akka.BaseActor
+
+import scala.collection.immutable.Seq
+
+sealed abstract class StoreFlags(prefix: String, silent: Boolean, flags: String*) {
```

Comment:
cannot you use something better than a boolean to express silent? It makes the test not very clear about the use of this boolean